db_stress_tool/db_stress_common.cc

```diff
@@ -280,7 +280,7 @@ void RemoteCompactionWorkerThread(void* v) {
       // Add the output regardless of status, so that primary DB doesn't rely on
       // the timeout to finish waiting. The actual failure from the
       // deserialization can fail the compaction properly
-      shared->AddRemoteCompactionResult(job_id, serialized_output);
+      shared->AddRemoteCompactionResult(job_id, s, serialized_output);
```

Comment:
Can you specify more in the benefit of doing this? I understand the change below with no timeout check but not this one yet.